pipeline/00-ingest.R

```diff
@@ -414,11 +293,10 @@ assessment_data_clean <- assessment_data_shifted %>%
     time_sale_day_of_month = as.integer(day(meta_sale_date)),
     time_sale_day_of_week = as.integer(wday(meta_sale_date)),
     time_sale_post_covid = meta_sale_date >= make_date(2020, 3, 15)
-  ) %>%
-  select(-time_interval)
```

Comment:
Removed since it's gone in res avm, assuming this is just removing a column that isn't used later in the pipeline?